PocketKit/Sources/PocketKit/Settings/AccountViewModel.swift

```diff
@@ -3,13 +3,15 @@ import Analytics
 import Textile
 import SharedPocketKit
 import SwiftUI
+import Combine
```

Comment:
what needs combine here?

Reply:
Nothing, it must be a leftover from something that I tried and decided not to do in the end 😄 . Thanks for the catch!